sphinx/conf.py

```diff
@@ -432,5 +432,6 @@
     r'https://www.nis-elements.com/.*', # Invalid SSL certificate
     r'https://www.nikoninstruments.com/.*', # Invalid SSL certificate
     r'http://farsight-toolkit.ee.uh.edu/.*',
-    r'https://testng.org/*' # Invalid SSL certificate
+    r'https://testng.org/*', # Invalid SSL certificate
```

Comment:
These are regexps, not shell globs, hence the `.*` above. But do we even need a trailing "anything" or do these match part of the URL instead of necessarily the whole? Might be worth trying omitting that last bit altogether then re-triggering.

Reply:
I suspect you are right, probably worth a test at some point to see